edb/pgsql/delta.py

```diff
@@ -4610,6 +4628,18 @@ def _create_property(
                     default_value = self.get_pointer_default(
                         prop, schema, context)
 
+                    if (
+                        isinstance(src.scls, s_links.Link)
+                        and not default_value
+                        and prop.get_default(schema)
+                    ):
+                        raise errors.UnsupportedFeatureError(
+                            f'default value for '
+                            f'{prop.get_verbosename(schema, with_parent=True)}'
+                            f' is too complicated; link property defaults '
```

Comment:
perhaps `default value for .. must be a constant expression`?

Reply:
Well, it can be stuff like `random()`, though